cli/commands/migrate_create.go

```diff
@@ -111,12 +123,21 @@ func (o *migrateCreateOptions) run() (version int64, err error) {
 	if o.flags.Changed("metadata-from-file") && o.sqlServer {
 		return 0, errors.New("only one sql type can be set")
 	}
+
 	if o.flags.Changed("metadata-from-file") && o.metaDataServer {
 		return 0, errors.New("only one metadata type can be set")
 	}
 
+	if o.flags.Changed("up-sql") && !o.flags.Changed("down-sql") {
```

Comment:
May be wrap this condition in a variable, something like
```
isSQLFromFlags :=  o.flags.Changed("up-sql") && o.flags.Changed("down-sql")
```
and then operating on that, thoughts ?